### PR TITLE
Support Command Line Tool as product type

### PR DESCRIPTION
**Pull request by nickkjordan**

### CODE REVIEW

File: lib/xcodeproj/constants.rb [hunk 1]
```diff
@@ -92,15 +92,16 @@ module Constants
...
-      :watch_app        => 'com.apple.product-type.application.watchapp',
-      :watch_extension  => 'com.apple.product-type.watchkit-extension',
+      :application       => 'com.apple.product-type.application',
+      :framework         => 'com.apple.product-type.framework',
+      :dynamic_library   => 'com.apple.product-type.library.dynamic',
+      :static_library    => 'com.apple.product-type.library.static',
+      :bundle            => 'com.apple.product-type.bundle',
+      :unit_test_bundle  => 'com.apple.product-type.bundle.unit-test',
+      :app_extension     => 'com.apple.product-type.app-extension',
+      :command_line_tool => 'com.apple.product-type.tool'
```
Comment (segiddins) [inline lib/xcodeproj/constants.rb]: this is missing the trailing comma and thus will cause a `SyntaxError`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Xcodeproj Changelog
 
+## Master
+
+#### Enhancements
+
+* `Constants`: Adds support for Command Line Tool as a product type
```
Comment (segiddins) [inline CHANGELOG.md]: Period and two trailing spaces, please :)

**Comment by nickkjordan** [comment]
Cool, thanks @segiddins !

**Comment by segiddins** [comment]
@neonichu this look right?

**Comment by neonichu** [comment]
LGTM :+1:

**Pull request merged**